p2p/3.IPFS_or_HTTP/README.md

```diff
@@ -0,0 +1,149 @@
+db_1   | 2021-08-02 20:48:37.283 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
+db_1   | 2021-08-02 20:48:37.286 UTC [25] LOG:  database system was shut down at 2021-08-02 20:48:32 UTC
+db_1   | 2021-08-02 20:48:37.289 UTC [1] LOG:  database system is ready to accept connections
+web_1  | No changes detected
+web_1  | Operations to perform:
+web_1  |   Apply all migrations: admin, auth, contenttypes, musicshare, sessions
+web_1  | Running migrations:
+web_1  |   No migrations to apply.
+web_1  | Watching for file changes with StatReloader
+web_1  | Performing system checks...
+web_1  | 
+web_1  | System check identified no issues (0 silenced).
+web_1  | August 02, 2021 - 20:48:39
+web_1  | Django version 3.1.6, using settings 'music_share_project.settings'
+web_1  | Starting development server at http://0.0.0.0:8000/
+web_1  | Quit the server with CONTROL-C.
+
+```
+As you can see, `db_1` is our database which seems okay and ready to accept new connections,
+and `web_1` is the backend of our website. You can open your favorite browser like Firefox or Chrome (or Opera,
+no discrimination here) and go to this url : [http://0.0.0.0:8000/](http://0.0.0.0:8000/).
+
+You should see our beautiful website !!
+
+
+If you want to shut down the server, use `Ctrl` + `C` and then run `docker-compose down`.
+
+### 1.1 Storage
+
+Go to [this](https://freemusicarchive.org/music/Scott_Holmes/rock-background-music/country-road-drive) website and download the song.
+By the way, you can look for any other genres of music you prefer on this website, it is free and open source. For the example, we are going to
+stick with this song.
+Go back to [http://0.0.0.0:8000/](http://0.0.0.0:8000/), scroll down and click on the button `upload a song`.
+Fill the form correctly and validates it. 
+
+You can find the code of the form in `./musichare/forms.py`, and the associate model object in `./musicshare/models.py`.
+
+Now, look at your terminal, you should see those strange logs appear:
+```bash
+web_1  | [06/Aug/2021 11:02:33] "GET / HTTP/1.1" 200 4650
+web_1  | [06/Aug/2021 11:02:37] "GET /upload/ HTTP/1.1" 200 4640
+web_1  | [06/Aug/2021 11:02:57] "POST /upload/ HTTP/1.1" 302 0
+web_1  | [06/Aug/2021 11:02:57] "GET /success/ HTTP/1.1" 200 4360
+web_1  | [06/Aug/2021 11:02:59] "POST / HTTP/1.1" 200 5120
+web_1  | [06/Aug/2021 11:02:59] "GET /media/static/Scott_Holmes_Music_-_Country_Road_Drive.mp3 HTTP/1.1" 200 8983053
+```
+Let me explain:
+the first part of the message is obviously the datetime. The second one is the method, `GET`, because we want to just
+get the page ; we are asking the server to give us the `/` route which is the home page.
+Then, the `HTTP` protocol and finally two numbers. The `200` is the most interesting : it is a status, preview code. `200` means
+that the server is ok to give us that page from the `/` route, and it has been delivered correctly.
```

Comment:
Maybe a [flowchart](https://en.wikipedia.org/wiki/Flowchart) you'll be more accurate